package.json

```
@@ -2,20 +2,30 @@
   "name": "@jahia/security-filter",
   "version": "2.0.0",
   "scripts": {
-    "dev": "./node_modules/.bin/webpack --watch --inline",
-    "build": "./node_modules/.bin/webpack --config=webpack.build.config.js"
+    "dev": "yarn webpack --watch",
+    "build": "yarn webpack"
```

Can't we have simply 
```suggestion
    "dev": "webpack --watch",
    "build": "webpack"
```
?